src/Resources/config/serializer/Symfony.Component.Messenger.Stamp.SerializerStamp.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Symfony\Component\Messenger\Stamp\SerializerStamp" exclusion-policy="ALL">
```

Comment:
Don't think we need this one, this bundle provides serializer stamps?

Reply:
Yeah, you are right. Sorry, will take it out.